### PR TITLE
Add turning-off of parser for languages with spaces in recommended settings

### LINKED ISSUE #1573: Add turning-off internal yomitan parser to languages with spaces to recommended settings
A parser's not needed for languages with spaces

image

Let's make this a recommended setting

### PR DESCRIPTION
**Pull request by liuwllm**
Closes #1573 

Adds two settings to languages with spaces:
- Set parsing.enableScanningParser to false to turn off internal parser
- Set parsing.termSpacing to false to remove added spaces between parsed words

### CODE REVIEW

**Comment by Kuuuube** [comment]
What does termspacing do when the parser is turned off?

**Comment by liuwllm** [comment]
Realizing this is a bit redundant if the space is only added to parsed words anyway; will add another commit to remove this from recommended settings

**Review by Kuuuube — APPROVED** [review]

File: ext/data/recommended-settings.json [hunk 1]
```diff
@@ -203,6 +305,54 @@
...
+    "ko": [
+        {
+            "modification": {
+                "action": "set",
+                "path": "parsing.enableScanningParser",
+                "value": false
+            },
+            "description": "Turn off Yomitan's internal parser for languages with spaces."
+        }
+    ],
```
Comment (Kuuuube) [inline ext/data/recommended-settings.json:327]: Is this a mistake?

**Review by Kuuuube — CHANGES_REQUESTED** [review]

File: ext/data/recommended-settings.json (hunk 1 above)
Comment (Kuuuube) [reply ext/data/recommended-settings.json:327]: Nvm korean characters look kindof like the space is part of the character or font and tricked me.

**Review by Kuuuube — APPROVED** [review]

**Pull request merged**